__tests__/app.test.js

```diff
@@ -103,6 +103,7 @@ describe("App API, /api", () => {
                 title: "Living in the shadow of a great man",
                 topic: "mitch",
                 votes: 100,
+                comment_count: 11,
```

Comment:
This is quite a brittle way of testing. Ideally, we would have a new test block for the additionally added value. Therefore something like expect object.containing would be far more flexible, as this will check that those values are present, but won't break it if there are more values added.

Reply:
Sorry not sure I understand. You mean to refactor the test so that I test for the key and variable type of the value for get articles then a another test for the key comment_count?